main.tex

```diff
@@ -626,12 +626,24 @@ \subsection*{Prediction of resistance phenotypes for novel non-synonymous varian
 
 %%
 %
-...with the evidence for selection on Anopheles haplotypes, knowledge of the gene structure and evidence from other systems we can begin to predict resistance phenotypes for novel non-synonymous variants...
+Both indirect (clustering, networks) and direct evidence (EHH) for selection on \textit{Vgsc} haplotypes we detected, can be used in concert with knowledge of the gene's structure, linkage of variants, and evidence of phenotypes from both \textit{Anopheles} and other insect systems, to predict putative resistance phenotypes for the novel non-synonymous variants detected.
+%
+These predictions could help define relationships between newly detected variants and prioritise assay design for molecular tracking of variants/haplotypes in the field.
+%
+To explore this potential for predication, we divide the resistance phenotypes into known (experimentally shown) and putative, and divide the causative variants into phenotype drivers or enhancers.
+%
+Driver variants occur on haplotypes first, initiating the resistance phenotype and starting the selective sweep. 
+%
+Experimentally established resistance variants \texttt{L995F}, \texttt{L995S} in \textit{An. gambiae} and \texttt{V402L} in \textit{A. aegypti} (\textit{An. gambiae} codon numbering) are examples of known driver variants (Table \ref{table:variants_pheno}). 
+%
+Enhancer variants occur on haplotypes already undergoing selective sweeps due to driver variants, these enhancer variants either increase the resistant phenotype above that of the driver variants alone, or ameliorate fitness costs of carrying a driver. 
+%
+The \texttt{N1570Y} variant, which has only been found on haplotypes carrying \texttt{L995F} and, using \textit{Xenopus} oocytes, has been shown to produce an increased resistance phenotype only when paired with \texttt{L995F} \cite{Wang2015}, is thus classed as an enhancer variant (Table \ref{table:variants_pheno}). 
+%
```

Comment:
Having now read Wang 2015, I think we need to make this discussion sub-section more nuanced. I'll have a crack at drafting something and post back here.